dev/drivers/scripts/mesoscale/plots/jevs_mesoscale_grid2obs_plots.sh

```diff
@@ -15,12 +15,12 @@ module reset
 export machine=WCOSS2
 
 # ECF Settings
-export RUN_ENVIR=nco
 export SENDECF=YES
 export SENDCOM=YES
 export KEEPDATA=NO
 export SENDDBN=YES
```

Comment:
@RoshanShrestha-NOAA On this line and in all of your dev/drivers/scripts, can you please set export SENDDBN=NO? We don't actually want to send DBN alerts in the emc.vpppg parallel. That's only for ops. Thanks!